src/uniprotkb/components/statistics/HistoricalReleasesEntriesLinePlot.tsx

```diff
@@ -0,0 +1,113 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import {
+  select,
+  scaleLinear,
+  axisBottom,
+  axisLeft,
+  line,
+  scaleTime,
+  format,
+  timeFormat,
+} from 'd3';
+
+import styles from './styles/historical-release-entries-line-plot.module.scss';
+
+// Specify the chart’s dimensions.
+const width = 400;
+const height = 400;
+const margin = { top: 20, right: 60, bottom: 45, left: 80 };
+
+export type DateCount = [Date, number];
+export type Bounds = {
+  date: [Date, Date];
+  count: [number, number];
+};
+
+type Props = {
+  dateCounts?: DateCount[];
+  bounds?: Bounds;
+};
+
+const HistoricalReleasesEntriesLinePlot = ({ dateCounts, bounds }: Props) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  const renderPlot = useCallback((dateCounts: DateCount[], bounds: Bounds) => {
+    const chart = select(svgRef.current).select('g');
+
+    // x-axis
+    const xScale = scaleTime()
+      .domain(bounds.date) // units: Date
```

Comment:
```suggestion
      .domain([startYear, endYear]) // units: Date
```
Corresponding change in order to get the bounds to include the beginning of the first year of data, and the tick for the upcoming year after the current last data point